R/postprob.R

```diff
@@ -81,6 +88,15 @@ postprob <- function(x, n, p, parE = c(1, 1), weights, betamixPost, log.p = FALS
     if (missing(weights)) {
       weights <- rep(1, nrow(parE))
     }
+    if (sum(weights) != 1) {
```

Comment:
this is not the right place, the weights checking and normalization happens in `h_getBetamixPost`. If you like you can put a warning in there instead.